projects/plugins/jetpack/modules/contact-form.php

```diff
@@ -30,7 +30,7 @@
  *
  * @param bool $load_contact_form_package Load Jetpack Forms package. Default to false.
  */
-if ( apply_filters( 'jetpack_contact_form_use_package', false ) ) {
+if ( apply_filters( 'jetpack_contact_form_use_package', true ) ) {
```

Comment:
Not sure if there's a way we can make the filter always return true by default, otherwise we'll also need to change the default value here: https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/jetpack/class.jetpack-gutenberg.php#L695

The line above allows us to control whether we want to register the blocks from the jetpack plugin or the blocks from the Forms package.

Reply:
Might turn into a race condition. I think we should add somewhere at the top `add_filter( 'jetpack_contact_form_use_package', '__return_true' );`, but it would also be legal to get that line inside this `if`, so, if nobody has a saying on it, we'll enable it for all subsequent calls:

```
if ( apply_filters( 'jetpack_contact_form_use_package', true ) ) {
  Jetpack_Forms::load_contact_form();
  add_filter( 'jetpack_contact_form_use_package', '__return_true' );
  return true;
}
```

Reply:
Not sure I understand what you mean 🤔 Wouldn't simply changing the other line you linked to to `true` as well so it's consistent fix the issue?